GiveNGo/GiveNGo
# Chat Page

## Description:
- Changes made include the addition of a **ChatPage** to display all currently open chats between users. The current iteration includes the ability to scroll through your chats, and swipe to delete messages. Additionally, `<Button />` and `<Avatar />` icons have been added to each message. Each `ListItem` component is rendered to the ChatPage with onPress events to route users to a separate ChatFeed page (under construction).

## Changes I Made:
- Created MockData for each `ListItem `component rendered to our **ChatPage**
- Update screen view to be **scrollable** and contain a series of List Items, each with **onPress** events and delete Buttons
- Incorporate **Stylesheet.create** hook to generate new styling options

## How to Test:
- `npm start`
- Navigate from the **Login** page to the **Chat** page from the **Header** navigator 
- Chat page should render list items with default data provided

![Screen Shot 2020-07-22 at 12 05 23 PM](https://user-images.githubusercontent.com/55554784/88217593-a4d3a480-cc13-11ea-81bc-0fd7807fab13.png)
